### PR TITLE
Replacing Image with Base Image

### PR DESCRIPTION
**Pull request by pavankumarj**
# Description

- Currently we are not able to extend Image functionality to support extensibility features like Auth, SVG support etc, this change make it possible to override BaseImage, which will give the developer the control to add additional feature set.
- We have created an internal registry to handle components that are not part of the schema
- Updated Readme.md to provide an example for overriding existing internal component

# How is the change verified

How you verified the fix, including one or all of the following:
1. All the existing image scenarios are verified via Visualizer

# Design Doc
[Common image component.docx](https://github.com/BigThinkcode/AdaptiveCards/files/8040511/Common.image.component.docx)

# Validated Screens (Visualizer)
Screenshot_20220210-150306_RNAdaptiveCards
Screenshot_20220210-150314_RNAdaptiveCards

Screenshot_20220210-150350_RNAdaptiveCards
Screenshot_20220210-150350_RNAdaptiveCards
Screenshot_20220210-150350_RNAdaptiveCards
Screenshot_20220210-150210_RNAdaptiveCards
Screenshot_20220210-150333_RNAdaptiveCards
Screenshot_20220210-150142_RNAdaptiveCards
Screenshot_20220210-150125_RNAdaptiveCards
Screenshot_20220210-150149_RNAdaptiveCards
Screenshot_20220210-150439_RNAdaptiveCards

### CODE REVIEW

File: source/community/reactnative/README.md [hunk 1]
```diff
@@ -65,6 +69,11 @@ Registry.getManager().registerComponent('Rating',RatingComponent);
 Registry.getManager().removeComponent('Input.Date');
 ```
 
+* To override an internal the support of an element type (i.e To ignore the rendering of specific element type),
```
Comment (vivekkozhisseri) [inline source/community/reactnative/README.md:68]: How this is different from the above line? It looks repeating for me.

**Review by vivekkozhisseri — DISMISSED** [review]
code LGTM. I have a small comment on the `readme`. Please have a look.

File: source/community/reactnative/README.md (hunk 1 above)
Comment (pavankumarj) [reply source/community/reactnative/README.md:68]: resolved

**Comment by pavankumarj** [comment]
@pragadeeshk I have updated documentation, please re-approve the PR

**Review by pragadeeshk — APPROVED** [review]
LGTM

**Pull request merged**